doc/en/user/source/community/gwc-mbtiles/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _community_gwc_mbtiles:
+
+GWC MBTiles layer plugin
+========================
+
+This plugin allows adding a GWC MBTiles Layer to GeoServer's cached Tile Layers configuration so that GeoServer can load cached tiles from ready-to-use MBTiles.
+
+Installation
+------------
+
+As a community module, make sure you have downloaded the :download_extension:`gwc-mbtiles`
+
+To install the module, unpack the zip file contents into the GeoServer ``WEB-INF/lib`` directory and restart GeoServer.
+
+Configuring a MBTilesLayer
+--------------------------
+The configuration of a new MBTilesLayer can be done by editing the geowebcache configuration file located in :file:`<data_dir>/gwc/geowebcache.xml`
+
+Locate the `layers` section of the config or add it if missing.
+
+Then define a new `mbTilesLayer` node for each new MBTilesLayer you want to add. A GWC configuration containing a configured MBTiles Layer would look like this:
+
+
+.. code-block:: xml
+
+   <gwcConfiguration>
+     <!-- ... -->
+     <layers>
+       <!-- ... -->
```

Comment:
A minimal but valid configuration for the layer would be helpful. GeoServer users are not used to play with GWC config files, giving them a "copy, paste and customize" approach is going to reduce confusion and questions.